### PR TITLE
GitHub: Add hero image

### PR DESCRIPTION
**Pull request by Lamparter**

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -32,6 +30,9 @@
       <source media="(prefers-color-scheme: light)" srcset="https://get.microsoft.com/images/en-us%20dark.svg" width="200" />
       <img src="https://get.microsoft.com/images/en-us%20light.svg" width="200" />
     </picture></a>
+  <a title="GitHub" href="https://github.com/0x5BFA/FluentHub/releases/latest">
```
Comment (0x5bfa) [inline .github/README.md:33]: I don't think this is necessary.

**Review by 0x5bfa — CHANGES_REQUESTED** [review]
Can you also update screenshots?

File: .github/README.md [hunk 2]
```diff
@@ -12,11 +12,9 @@
...
-  <img width="128" align="center" src="../assets/fluenthub.png" />
+  <img alt="FluentHub hero image" src="/assets/header.png" />
 </p>
-<h1 align="center">
-  FluentHub
-</h1>
+
 <p align="center">
   <a style="text-decoration:none" href="https://github.com/0x5bfa/FluentHub/actions/workflows/ci.yml">
     <img src="https://github.com/0x5bfa/FluentHub/actions/workflows/ci.yml/badge.svg" alt="CI Status" /></a>
```
Comment (Lamparter) [inline .github/README.md:20]: This section looks a bit weird now that we added the header image.
@0x5bfa should we copy how it is in Files with the download buttons?

Comment (0x5bfa) [reply .github/README.md:20]: I don't think so this comes from MS Store and I made the banner before they made.

**Review by 0x5bfa — APPROVED** [review]

File: .github/README.md (hunk 2 above)
Comment (Lamparter) [reply .github/README.md:20]: > I don't think so this comes from MS Store and I made the banner before they made.

No I just meant it looks a bit weird with the Store button being where it is.... nevermind...

**Pull request merged**